_includes/search.html

```diff
@@ -0,0 +1,4 @@
+<input placeholder="Search…" type="search" id="search" class="grid-block shrink search">
+<div role="dialog" class="grid-block vertical popover popover-search margin-left-sm margin-right-sm lunr-hidden" id="results">
+</div>
+<div id="overlay" class="lunr-hidden"></div>
```

Comment:
<img width="385" alt="screen shot 2017-05-23 at 6 46 53 pm" src="https://cloud.githubusercontent.com/assets/5341618/26383547/36bf8d0c-3fe8-11e7-84e6-7a5b08e16202.png">

Even if we want a fixed height, this height is a bit excessive?